src/library_webgl.js

```diff
@@ -4279,6 +4279,10 @@ function recordGLProcAddressGet(lib) {
   Object.keys(lib).forEach((x) => {
     if (x.startsWith('gl') && !isDecorator(x)) {
       lib['emscripten_' + x] = x;
+      var sig = LibraryManager.library[x + '__sig'];
```

Comment:
how is this part related to the rest of it?

Reply:
This lack of correct signatures for `emscripten_glXXXX` wrappers was causing the test to fail.